### DISCUSSION TITLE
New translation

### DISCUSSION THREAD

**Discussion by mwxp06** [Koenkk/zigbee2mqtt#9741] [General]
Hallo,
is possible to add new language translation?

I would like to translate frontend to Czech language.

Thank you

**Comment by sjorge** [comment]
IIRC the translation files live here: https://github.com/nurikk/zigbee2mqtt-frontend/tree/dev/src/i18n/locales

**Reply by mwxp06** [reply]
Thank you, I will add new translation there.
